server/base/src/main/java/org/apache/accumulo/server/ServerContext.java

```diff
@@ -433,4 +435,13 @@ private void monitorSwappiness(AccumuloConfiguration config) {
       }
     }, 1000, 10 * 60 * 1000, TimeUnit.MILLISECONDS);
   }
+
+  public ScheduledThreadPoolExecutor getSharedGenericScheduledExecutorService() {
```

Comment:
Can sync to avoid race condition of multiple thread creating when its null.  Also I think a shorter name would be nice, withe the fact that its shared moved to javadoc.

```suggestion
  /**
   * return a shared scheduled executor
   */
  public synchronized ScheduledThreadPoolExecutor getScheduledExecutor() {
```